src/styles/Css/App.css

```diff
@@ -21,74 +20,198 @@ body {
 body h2,
 body span,
 body h3 {
-  color: #03045e;
+  color: #121212;
   font-weight: 600;
   font-size: 1em;
 }
 
+.bookstoreTitle,
+.fullWebsite,
+.CategoriesComponent,
+.ContactComponent,
+.App,
+.BooksContainer,
+.BookDetails,
+.BookChapter,
+.Books {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
 button {
   transition: all 0.5s;
   border-radius: 2px;
-  color: #03045e !important;
```

Comment:
- [ ] Kindly always follow the best practices in CSS or style. You are using the `important` keyword, and your previous code reviewer pointed out this issue, which actually changes the normal flow of the `paint` of the browser. Because `important` take super priority over other style and also can create an issue while working on the media queries. So that's why it is not a good idea and also against the requirements of the project.

### Note:

Please remove the `important` keyword entirely from your CSS.